pom.xml

```diff
@@ -41,7 +41,7 @@
   </distributionManagement>
 
   <properties>
-    <nxrm-version>3.19.0-01</nxrm-version>
```

Comment:
Q: Is this to be updated to an actual build once we release 3.21?  Is there some trigger to help remind (like 3.next)?

Reply:
We do not have any triggers for community formats. I have just wanted to up parent version to the latest